app/src/controllers/pages/actionCreators.js

```diff
@@ -19,5 +18,5 @@ export const updatePagePropertiesAction = (properties) => (dispatch, getState) =
     meta: { query: newQuery },
   };
 
-  dispatch(redirect(updatedAction));
+  dispatch(updatedAction);
```

Comment:
Thet's tricky. For example, if I add/remove different filters, by pressing 'Back' button I will go through the history of my filter edits. I don't think it's convenient since there may be many edits, and in this case I won't get back to the previous page.
I think this behavior should be discussed with BA.

Reply:
I've discussed this behavior with BA. She approved it. Changing query should affect browser history.